From 4016086ce17bc692ea05fd0d5fdef223f6cf67b6 Mon Sep 17 00:00:00 2001
From: root <root@localhost.localdomain>
Date: Wed, 26 Feb 2020 21:47:03 +0800
Subject: [PATCH] plus for us

---
 readme.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/readme.txt b/readme.txt
index 46d49bf..8a6ded7 100644
--- a/readme.txt
+++ b/readme.txt
@@ -1,2 +1,2 @@
 Git is a version control system.
-Git is free software.
+Git is free software for us.
-- 
2.22.0.windows.1

